examples/examples/thread-info.rs

```diff
@@ -0,0 +1,43 @@
+#![deny(rust_2018_idioms)]
+/// This is a example showing how thread info can be enabled
+/// to be attached with events
```

Comment:
How about:
```suggestion
/// This is a example showing how thread info can be displayed when
/// formatting events with `tracing_subscriber::fmt`.
```

Reply:
I believe that this may have been incorrectly resolved; I like @hawkw's proposed change.

Reply:
Ahhh, I had this locally but did not save it. Thank you! :)